Project-Descriptions-and-Plans/CVA6/CVA6-plan-approved-2023-2024.md

```diff
@@ -0,0 +1,420 @@
+| Core            | FPGA optimizations: focus on frequency increase                                                                                | 2023-Q4       | TRT                                             |
+| Core            | Corrections related to verification activities                                                                                 | Continuous    | TSS and MU-Electronics                          |
+| Verification    | Verification step 1: DVPlans                                                                                                   | 2023-Q3       | TSS (lead), 10xEngineers, MU-Electronics, ECC   |
+| Verification    | Verification step 1: CV32A60X is verified at programmer's view level                                                           | 2023-Q4       | TSS (lead), 10xEngineers, MU-Electronics, ECC   |
+| Verification    | Verification step 1: Create Spike tandem                                                                                       | 2023-Q3       | TSS (lead)                                      |
+| Verification    | Verification step 2: DVPlans                                                                                                   | 2024-Q3       | TSS (lead), 10xEngineers, MU-Electronics, ECC   |
+| Verification    | Verification step 2: CV32A60X is fully verified (programmer's view level + microarchitecture)                                  | 2024-Q4       | TSS (lead), 10xEngineers, MU-Electronics, ECC   |
+| Verification    | CV32A60X released, Project Freeze (PF) checklist                                                                               | 2024-Q4       | TSS (lead) and verification team                |
+| Software        | Linux Yocto (with BuildRoot and OpenSBI) available in an OpenHW repository                                                     | 2023-Q2       | TRT                                             |
+|                 | _For information: FreeRTOS project related to CVA6_                                                                            |               |                                                 |
+| Related project | Validate PMP features on CVA6 with applications on FreeRTOS                                                                    | November 2023 | ECC                                             |
+| Related project | Integrate updated release of FreeRTOS kernel into CVA6 FreeRTOS with device drivers implemented, upstream to main repo         | TBD           | ECC                                             |
+| Related project | Port the new FreeRTOS on CVA6-apu                                                                                              | TBD           | ECC                                             |
+
```

Comment:
Since we are already in Q3, can you indicate if the 2023-Q2 items have already been achieved?

Reply:
Rejected. I want to keep it to a baseline.